python/ray/data/tests/test_logging.py

```diff
@@ -148,20 +150,22 @@ def test_json_logging_configuration(
         log_contents = file.read()
 
     # Validate the log is in JSON format (a basic check for JSON)
-    assert all(
-        log_line.startswith("{") and log_line.endswith("}")
-        for log_line in log_contents.splitlines()
-    )
+    messages = set()
+    for log_line in log_contents.splitlines():
+        log_dict = json.loads(log_line)  # will error if not a json line
+        messages.add(log_dict["message"])
```

Comment:
let's use a list and append to it, so that we avoid accidentally de-duplicating logs with the set. i don't think there's an issue with the current implementation / test, just want to future proof.